test/functional/basic/file-movement-e2e-specs.js

```diff
@@ -35,8 +36,8 @@ if (!process.env.REAL_DEVICE) {
         });
 
         it('should be able to fetch the Address book', async function () {
-          let file = 'Library/AddressBook/AddressBook.sqlitedb';
-          let stringData = await pullFileAsString(file);
+          let file = '../../../../../Library/AddressBook/AddressBook.sqlitedb';
```

Comment:
@jlipps I think you've found a nice security hole there, which allows to read/write any file on the server. We push/pull methods should contain a security verification, which confirms that absolute path is a subpath of the corresponding container.